ocaml-lsp-server/src/workspace_symbol.ml

```diff
@@ -0,0 +1,216 @@
+open Import
+
+let to_loc (loc : Warnings.loc) : Loc.t =
+  { loc_start = loc.loc_start
```

Comment:
Why is this needed? I'm pretty sure that `Warnings.loc = Loc.t`

Reply:
Good catch. That's something I needed in the first implementation that was using `ocp-index`